samples/MvcSandbox/Startup.cs

```diff
@@ -16,7 +16,9 @@ public class Startup
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc().AddCookieTempDataProvider();
+            services.AddMvc()
+                .AddCookieTempDataProvider()
+                .WithRazorPagesRoot("/");
```

Comment:
We'll have to rename the file since `/Pages/Index.cshtml` with a `/Page` root would collide with the `Home/Index` controller action.